### ISSUE TITLE
Reactor Auto Mode

### ISSUE THREAD

**Issue by waywardgamer** [zeroneye/Powah#105]
Minecraft 1.16.1
Forge 32.0.108
Powah 2.2.0

While drawing power from a reactor the internal storage will never hit completely full meaning the Auto Mode never engages.  Stored energy always tops out at max capacity minus however much you are drawing per tick and you keep burning fuel to produce far more FE than you need. Effectively the feature never engages.  If you stop the power draw and let it actually hit max capacity the feature seems to work, but if you have Refined Storage or Applied Energistics you are never going to stop drawing a relatively small amount of power.

**Comment by telemenar** [comment]
I just fought with this. I had constant drain in the 3-4K FE/t range on a Niotic Reactor that could make 40k FE/t which was enough that the reactor never kicked into auto off mode.

I tried to solve this with a comparator to trigger a shutoff but that didn’t work due to #86 

In the end I did it by using a comparator on a Mekanism energy cube to turn off the cable coming out of the reactor if the energy cube was at signal 14 or 15, so that the reactor would get to full because the unless the drain on the cube was more than 1/16 the full capacity of the cube (which I think would max the reactor as well) then I stop draining the reactor.
